server/channels/api4/config_local.go

```diff
@@ -172,7 +172,7 @@ func localMigrateConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	auditRec := c.MakeAuditRecord("migrateConfig", audit.Fail)
+	auditRec := c.MakeAuditRecord(model.AuditEventMigrateConfig, audit.Fail)
```

Comment:
Should this event also have a local prefix?

Reply:
Since I'm trying to keep the _same name_ as the "event-string-names", do you think it would be okay to update its corresponding "event name"? Can it cause an issue if someone has set up a monitoring filter on these **existing** keys?